config/samples/instancemgr_v1alpha1_instancegroup_with_managed_policies.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: instancemgr.keikoproj.io/v1alpha1
+kind: InstanceGroup
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: hello-world
+spec:
+  provisioner: eks-cf
+  strategy:
+      type: rollingUpdate
+  eks-cf:
+    maxSize: 3
+    minSize: 1
+    configuration:
```

Comment:
did you forget to add the `managedPolicies`? ;)

Reply:
I did not push it and wanted to test it and add some dummy values for other params like Subnet and SG just to avoid any issues of exposing target account details..